microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -70,8 +70,12 @@ The following list provides update details:
 //==== New feature for use during installation here
 //can include a note about a change in base RHEL image
 
-//[id="microshift-4-15-support"]
-//=== Support
+[id="microshift-4-15-support"]
+=== Support
+
+[id="microshift-4-15-etcd"]
+==== Getting the etcd version
+Previously, you could not query for the etcd version included with {microshift-short}. Now, the `microshift-etcd version` command outputs the {microshift-short} version and the base version of the etcd database. See xref:../microshift_support/microshift-etcd.adoc#microshift-etcd[{microshift-short} etcd] for more information.
```

Comment:
I'd recommend updating the link text to be the actual title of the page/section that you're linking to. So "The etcd service" in this case it looks like.